### PR TITLE
feat(getToken): allow user to get token using user/pass

### PR DESCRIPTION
**Pull request by parkererickson-tg**

### CODE REVIEW

**Review by qe-tigergraph — CHANGES_REQUESTED** [review]
Unit Test: FAILURE, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/1022/

File: pyTigerGraph/pyTigerGraphAuth.py [hunk 1]
```diff
@@ -241,16 +247,21 @@ def getToken(self, secret: str, setToken: bool = True, lifetime: int = None) ->
             except Exception as e:
                 raise e
 
-        if not res["error"]:
+        
+        if not res.get("error"):
+            print(res)
```
Comment (billshitg) [inline pyTigerGraph/pyTigerGraphAuth.py:252]: Is this print intended?

Comment (parkererickson-tg) [reply pyTigerGraph/pyTigerGraphAuth.py]: Nope. Removed

**Review by qe-tigergraph — COMMENTED** [review]
Unit Test: SUCCESS, e2e Test: SUCCESS, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/1028/

**Review by billshitg — APPROVED** [review]

**Pull request merged**